CMakeModules/FindOpenCL.cmake

```diff
@@ -72,10 +39,11 @@ function(_FIND_OPENCL_VERSION)
   CMAKE_PUSH_CHECK_STATE()
   foreach(VERSION "2_0" "1_2" "1_1" "1_0")
     set(CMAKE_REQUIRED_INCLUDES "${OpenCL_INCLUDE_DIR}")
+
     if(APPLE)
       CHECK_SYMBOL_EXISTS(
         CL_VERSION_${VERSION}
-        "${OpenCL_INCLUDE_DIR}/OpenCL/cl.h"
+        "${OpenCL_INCLUDE_DIR}/Headers/cl.h"
```

Comment:
Apple, really, `Headers`!

Reply:
This is the latest FindOpenCL.cmake from CMake v3.9. I just copied it here.